templates/dialogs/fight.hbs

```diff
@@ -74,32 +104,38 @@
         </div>
         {{#each participants as |p i|}}
         <div class="participant-card">
-            {{#if ../gmView}}
-            <i data-action="removeFighter" data-index="{{i}}" class="fas fa-times remove-fighter"></i>
-            {{/if}}
-            <div class="actor-name clickable" data-action="toggleHidden" data-index="{{i}}">{{p.name}}</div>
+            <div class="fighter-action-panel">
+                {{#if ../gmView}}
+                <i data-action="removeFighter" data-index="{{i}}" class="fas fa-times remove-fighter"></i>
+                {{/if}}
+                <div class="actor-name clickable" data-action="toggleHidden" data-index="{{i}}">{{p.name}}</div>
+            </div>
             {{#if p.showActions}}
-            <img class="portrait clickable" src="{{p.imgSrc}}" title="{{p.name}}" data-action="openSheet" data-actor-id="{{p.id}}"/>
             <div class="details">
-                <div>
-                    Reflexes: {{p.reflexes}}
-                </div>
-                <div>
-                    Engagement Bonus
-                    <input data-index="{{i}}" name="engagementBonus" type="number" class="exponent no-arrows" value="{{p.engagementBonus}}">
+                <div class="details-key">Reflexes</div>
+                <div class="details-value">{{p.reflexes}}</div>
+                <div class="details-key">Engagement</div>
+                <div class="withLabels details-value">
+                    <div class="exponent-meaning">D</div>
```

Comment:
`exponent-meaning` doesn't seem to appear anywhere, can be deleted.

Reply:
moreover, it's not clear to me why these appear in reverse order other than `flex-direction: row-reverse` is used in the `details-value` css class.